test/jdk/java/awt/event/HierarchyEvent/AncestorResized/AncestorResized.java

```diff
@@ -61,8 +54,7 @@ public void ancestorResized(HierarchyEvent ce) {
         }
     }
 
-    private static void init()
-    {
+    public static void main(String[] args) throws Exception {
         Frame frame;
```

Comment:
Not a big fan of the split declaration and initialization of all the variables but that existed before the fix and it is purely cosmetic. Other than that looks good.